### PR TITLE
Article on ackermann function

### LINKED ISSUE #426: Article on Ackermann Function
## 📚 Documentation

Article on Ackermann Function
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by satcasm**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Article on ackermann function

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #426

### CODE REVIEW

**Comment by satcasm** [comment]
Hey @HarshCasper , made the desired changes asked  in #439. Have a look.
Apologies for the inconvenience.

File: docs/mathematics/ackermann-function.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+A(i, j)=A(i-1, A(i, j-1)) for i, j > 0
+```
+Let's have few examples: 
+```
+For i=1,j=2
+Output=4
+
+For i=3,j=3
+Output=61
+```
```
[the remark this answers is not in the document]
Comment (satcasm) [reply docs/mathematics/ackermann-function.mdx]: okay

**Comment by satcasm** [comment]
@HarshCasper made the changes have a look.
Thanks!

**Comment by satcasm** [comment]
@HarshCasper is there something else that needs to be done?

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]

**Pull request merged**